### ISSUE TITLE
Add Profile: 

### ISSUE THREAD

**Issue by PRATHAM777P** [recodehive/awesome-github-profiles#1478]
<table align="center" border="1" cellspacing="0" cellpadding="10" style="border-collapse: collapse; width: 80%;">
  <tr>
    <td align="center">
      <img 
        src="https://github.com/user-attachments/assets/d60cb7d9-c445-4617-ae82-6b36904897bf" 
        alt="Coding - Pixel Jeff" 
        width="100%" 
        style="display: block; margin: 0 auto;" 
      />
    </td>
  </tr>
</table>

![](https://komarev.com/ghpvc/?username=PRATHAM777P&base=0)

<h1 align="center">
  <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" width="30px" />
  <span style="background: linear-gradient(to right, #00FFFF, #8A2BE2, #00FFFF); -webkit-background-clip: text; color: transparent; font-weight: 700;">
    Hi there, <img src="https://raw.githubusercontent.com/SP-XD/SP-XD/refs/heads/main/images/Developer.gif" height="30px" width="30px"> I'm Prathamesh Penshanwar
      </span><br/>
  <sub><em>(aka Pratham)</em></sub>
</h2>
  </span>

  
---

<h3 align="left">Connect with me:</h3>
<p align="left">
<a href="https://github.com/PRATHAM777P" target="_blank">
  <img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/github.svg" alt="PRATHAM777P" height="30" width="40" />
</a>
<a href="https://linkedin.com/in/prathamesh-penshanwar777p/" target="blank">
  <img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg" alt="prathamesh-penshanwar777p" height="30" width="40" /></a>
<a href="https://www.instagram.com/not_pratham_8/" target="blank">
  <img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/instagram.svg" alt="not_pratham_8" height="30" width="40" /></a>
<a href="https://hackerrank.com/prathamesh777h_r" target="blank">
  <img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/hackerrank.svg" alt="prathamesh777h_r" height="30" width="40" /></a>
<a href="https://leetcode.com/u/Prathamesh7exe/" target="blank">
  <img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/leet-code.svg" alt="Prathamesh7exe" height="30" width="40" /></a>
<a href="https://www.codechef.com/users/prath_7codex" target="blank">
  <img align="center" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/codechef.svg" alt="prath_7codex" height="30" width="40" /></a>
<a href="https://www.hackerearth.com/@prathameshpenshanwar777" target="blank">
  <img align="center" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/hackerearth.svg" alt="prathameshpenshanwar777" height="30" width="40" /></a>
<a href="https://auth.geeksforgeeks.org/user/prathameshpev8ob" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/geeks-for-geeks.svg" alt="prathameshpev8ob" height="30" width="40" /> </a>
<a href="https://kaggle.com/prathameshpenshanwar" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/kaggle.svg" alt="prathameshpenshanwar" height="30" width="40" /></a>

---
## <img src = "https://github-production-user-asset-6210df.s3.amazonaws.com/73993775/283929614-7d083e4b-8c04-4c94-b996-085e97c9a6a0.gif" width = 41px>&nbsp;&nbsp; About Me

<img align = "right" src = "https://github-production-user-asset-6210df.s3.amazonaws.com/73993775/283930112-76687f51-ac99-48ef-a44b-dd11c1d78e7f.gif" width = 250px></img>

Junior AI/ML Engineer and E&TC (Electronics & Telecommunication) Engineer passionate about building intelligent, real-world solutions.

I work across:

- 🧠 AI (Artificial Intelligence)  
- 📊 ML (Machine Learning)  
- 🌐 IoT (Internet of Things)  
- 📱 Android Development  

to create smart and impactful applications, with a strong focus on innovation and problem-solving.

- 🗣️ **Languages:** 🇬🇧 English | 🇮🇳 Marathi | 🇮🇳 Hindi
---

## 🧰 My Tech Stack

<table>
  <tr>
    <th>Programming Languages</th>
    <th>IDEs / Editors</th>
    <th>Frameworks / Libraries</th>
    <th>Databases / Query Languages</th>
    <th>Platforms / Tools</th>
    <th>Game Engines</th>
  </tr>
  <tr>
    <td align="center">
      <img src="https://skillicons.dev/icons?i=python&theme=light" width="32px" title="Python"/>
      <img src="https://skillicons.dev/icons?i=cpp&theme=light" width="32px" title="C++"/>
      <img src="https://skillicons.dev/icons?i=kotlin&theme=light" width="32px" title="Kotlin"/>
      <img src="https://img.shields.io/badge/SQL-336791?style=flat&logo=postgresql&logoColor=white" height="32px" title="SQL (PostgreSQL)"/>
    </td>
    <td align="center">
      <img src="https://skillicons.dev/icons?i=vscode&theme=light" width="32px" title="VS Code"/>
      <img src="https://skillicons.dev/icons?i=androidstudio&theme=light" width="32px" title="Android Studio"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/intellij/intellij-original.svg" width="32px" title="IntelliJ"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/pycharm/pycharm-original.svg" width="32px" title="PyCharm"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/matlab/matlab-original.svg" width="32px" title="MATLAB"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/jupyter/jupyter-original-wordmark.svg" width="32px" title="Jupyter"/>
    </td>
    <td align="center">
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/flask/flask-original.svg" width="32px" title="Flask"/>
      <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/pandas/pandas-original.svg" width="32px" title="Pandas"/>
      <img src="https://www.vectorlogo.zone/logos/pytorch/pytorch-icon.svg" width="32px" title="PyTorch"/>
      <img src="https://upload.wikimedia.org/wikipedia/commons/0/05/Scikit_learn_logo_small.svg" width="32px" title="Scikit-Learn"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/anaconda/anaconda-original.svg" width="32px" title="Anaconda"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/plotly/plotly-original-wordmark.svg" width="32px" title="Plotly"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/pytest/pytest-original-wordmark.svg" width="32px" title="Pytest"/>
      <img src="https://skillicons.dev/icons?i=opencv&theme=light" width="32px" title="OpenCV"/>
      <img src="https://img.shields.io/badge/numpy-F77601?style=flat&logo=numpy&logoColor=white" height="32px" title="NumPy"/>
      <img src="https://img.shields.io/badge/pyttsx3-TTS-green?style=flat" height="32px" title="pyttsx3"/>
      <img src="https://img.shields.io/badge/requests-HTTP-orange?style=flat" height="32px" title="requests"/>
    </td>
    <td align="center">
      <img src="https://skillicons.dev/icons?i=mysql&theme=light" width="32px" title="MySQL"/>
      <img src="https://www.vectorlogo.zone/logos/sqlite/sqlite-icon.svg" width="32px" title="SQLite"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/mongodb/mongodb-original-wordmark.svg" width="32px" title="MongoDB"/>
    </td>
    <td align="center">
      <img src="https://skillicons.dev/icons?i=git&theme=light" width="32px" title="Git"/>
      <img src="https://skillicons.dev/icons?i=github&theme=light" width="32px" title="GitHub"/>
      <img src="https://skillicons.dev/icons?i=docker&theme=light" width="32px" title="Docker"/>
      <img src="https://skillicons.dev/icons?i=azure&theme=light" width="32px" title="Azure"/>
      <img src="https://skillicons.dev/icons?i=aws&theme=light" width="32px" title="AWS"/>
      <img src="https://skillicons.dev/icons?i=linux&theme=light" width="32px" title="Linux"/>
      <img src="https://skillicons.dev/icons?i=windows&theme=light" width="32px" title="Windows"/>
      <img src="https://skillicons.dev/icons?i=cmake&theme=light" width="32px" title="CMake"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/arduino/arduino-plain-wordmark.svg" width="32px" title="Arduino"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/confluence/confluence-original-wordmark.svg" width="32px" title="Confluence"/>
      <img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/gradle/gradle-original-wordmark.svg" width="32px" title="Gradle"/>
      <img src="https://img.shields.io/badge/MS%20Office-D83B01?style=flat&logo=microsoft-office&logoColor=white" height="32px" title="MS Office"/>
    </td>
    <td align="center">
      <img src="https://www.vectorlogo.zone/logos/unity3d/unity3d-icon.svg" width="32px" title="Unity"/>
      <img src="https://raw.githubusercontent.com/kenangundogan/fontisto/036b7eca71aab1bef8e6a0518f7329f13ed62f/icons/svg/brand/unreal-engine.svg" width="32px" title="Unreal Engine"/>
    </td>
  </tr>
</table>

---

### 📊 GitHub Stats & Activity
<div align="center">
  <img height="158em" src="https://github-profile-summary-cards.vercel.app/api/cards/profile-details?username=PRATHAM777P&theme=radical">
  <img height="158em" src="https://github-profile-summary-cards.vercel.app/api/cards/stats?username=PRATHAM777P&theme=radical">
  <img height="160em" src="https://github-profile-summary-cards.vercel.app/api/cards/repos-per-language?username=PRATHAM777P&theme=radical">
  <img height="160em" src="https://github-profile-summary-cards.vercel.app/api/cards/most-commit-language?username=PRATHAM777P&theme=radical">
  <img height="160em" src="https://github-profile-summary-cards.vercel.app/api/cards/productive-time?username=PRATHAM777P&theme=radical&utcOffset=5.5">
</div>
<div align="center">
  <img src="https://user-images.githubusercontent.com/74038190/212284100-561aa473-3905-4a80-b561-0d28506553ee.gif" alt="Bottom Line" width="100%" />
</div>

<table>
  <tr>
    <td align="center">
      <img width="100%" src="https://streak-stats.demolab.com?user=PRATHAM777P&theme=react&border_radius=10&ring=ffffff&fire=ff6b6b&currStreakLabel=ffffff&sideLabels=ffffff&dates=ffffff&stroke=ffffff" alt="GitHub Streak" />
    </td>
    <td align="center">
     <a href="https://github.com/PRATHAM777P">
    <img height="190em" src="https://github-readme-stats-eight-theta.vercel.app/api/top-langs/?username=PRATHAM777P&langs_count=8&layout=compact&theme=nightowl&border_radius=10" alt="Top Languages" />
    </td>
  </tr>
</table>

<div align="center">
  <img src="https://user-images.githubusercontent.com/74038190/212284100-561aa473-3905-4a80-b561-0d28506553ee.gif" alt="Bottom Line" width="100%" />
</div>

<p align="center">
  <img src="https://github-readme-activity-graph.vercel.app/graph?username=PRATHAM777P&theme=react-dark"/>
</p>

<div align="center">
  <img src="https://user-images.githubusercontent.com/74038190/212284100-561aa473-3905-4a80-b561-0d28506553ee.gif" alt="Bottom Line" width="100%" />
</div

<p align="center">
  <img src="https://raw.githubusercontent.com/czl9707/gh-space-shooter/main/example.gif" alt="Space shooter contribution graph" />
</p>

<div align="center">
  <img src="https://user-images.githubusercontent.com/74038190/212284100-561aa473-3905-4a80-b561-0d28506553ee.gif" alt="Bottom Line" width="100%" />
</div>

---

### ⚡ Fun Fact

🎶 Coding with beats = Ultimate Productivity! 🎧  

💡 Let’s Connect & Innovate Together! 🌟
<div align="center">
  <img src="https://user-images.githubusercontent.com/74038190/212284115-f47cd8ff-2ffb-4b04-b5bf-4d1c14c0247f.gif" alt="Bottom Line" width="100%" />
</div>
